.github/workflows/pr.yaml

```diff
@@ -174,6 +175,15 @@ jobs:
     with:
       build_type: pull-request
       script: "ci/test_python_cudf.sh"
+  conda-python-cudf-imports-cpu-only:
```

Comment:
Please don't add a new CI job for an import test. Use the conda recipe tests here, which I enabled in #18197:

https://github.com/rapidsai/cudf/blob/cadbd1ea59c872fc62df959449b5c725eb63c5e2/conda/recipes/cudf/recipe.yaml#L119-L122

Reply:
@bdice Do you know what is the right way to invoke the command in a conda recipe tests scripts? I have tried 2 combinations and both fail.

Reply:
Please see http://rattler.build/latest/testing/ for examples of conda package tests.